packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1093,11 +1093,11 @@
       <shortDescription>No related items</shortDescription>
     </labels>
     <labels>
-      <fullName>quantic_OpensInSalesforceSubTab</fullName>
```

Comment:
The old name of the label seems good to me.

Reply:
ok this was to match the new label value more closely but since we keep 'Salesforce' I will revert it!